modules/blackhole-monitor/src/monitor.py

```diff
@@ -0,0 +1,116 @@
+import boto3
+import logging
+import json
+import os
+
+cloudwatch = boto3.client('cloudwatch')
+ec2 = boto3.client('ec2')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def lambda_handler(event, context):
+  # Get transit gateways
+  getTransitGateways = ec2.describe_transit_gateways()
+
+  # Loop over transit gateways
+  for i in getTransitGateways['TransitGateways']:
+    print("Transit Gateway: " + i['TransitGatewayId'])
+    # Loop over transit gateway route tables filtering on transit gateway ID
+    getTransitGatewayRouteTables = ec2.describe_transit_gateway_route_tables(
+      Filters=[
+        {
+            'Name': 'transit-gateway-id',
+            'Values': [
+                i['TransitGatewayId'],
+            ]
+        }
+      ]
+    )
+    # Search transit gateways
+    for j in getTransitGatewayRouteTables['TransitGatewayRouteTables']:
+      print("  Transit Gateway Route Table: " + j['TransitGatewayRouteTableId'])
```

Comment:
Swap `print` for `logger`

Reply:
https://github.com/ministryofjustice/hmpps-ems-platform-terraform-modules/pull/7/commits/1fb3fd60ac169b61a8244a6fda0cedab16c9b90a